bpf/lib/common.h

```diff
@@ -71,12 +71,13 @@
 #define CILIUM_CALL_IPV6_NODEPORT_NAT		16
 #define CILIUM_CALL_IPV4_NODEPORT_REVNAT	17
 #define CILIUM_CALL_IPV6_NODEPORT_REVNAT	18
-#define CILIUM_CALL_ENCAP_NODEPORT_NAT		19
-#define CILIUM_CALL_IPV4_NODEPORT_DSR		20
-#define CILIUM_CALL_IPV6_NODEPORT_DSR		21
-#define CILIUM_CALL_IPV4_FROM_HOST		22
-#define CILIUM_CALL_IPV6_FROM_HOST		23
-#define CILIUM_CALL_SIZE			24
+#define CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT	19
+#define CILIUM_CALL_IPV6_ENCAP_NODEPORT_NAT	20
+#define CILIUM_CALL_IPV4_NODEPORT_DSR		21
+#define CILIUM_CALL_IPV6_NODEPORT_DSR		22
+#define CILIUM_CALL_IPV4_FROM_HOST		23
+#define CILIUM_CALL_IPV6_FROM_HOST		24
+#define CILIUM_CALL_SIZE			25
```

Comment:
The corresponding change to the verifier complexity script wasn't included in this commit, was it later or somehow omitted?

Reply:
Omitted but not forgotten. I'm just checking complexity issues so far, hence the draft mode. I also suspect these few line changes are the reason for the downgrade test failure on net-next.